charts/incubator/teamspeak/values.yaml

```diff
@@ -105,6 +105,8 @@ metrics:
     env:
       # -- metrics port
       port: 9189
+      # -- TeamSpeak query endpoint
+      remote: localhost:10011
```

Comment:
I don't know the app, so I just want to make sure that this is a sensible default that will not break peoples existing installs?